PlayCover/Views/App Views/ClearCacheModalView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ClearCacheModalView.swift
+//  PlayCover
+//
+
+import SwiftUI
+import DataCache
+
+struct ClearCacheModalView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @Binding var app: PlayApp
+
+    @State private var isClearingCache = false
+    @State private var appIcon: NSImage?
+    @State private var cache = DataCache.instance
+
+    var body: some View {
+        VStack {
+            Group {
+                if let image = appIcon {
+                    Image(nsImage: image)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                } else {
+                    Rectangle()
+                        .fill(.regularMaterial)
+                        .overlay {
+                            ProgressView()
+                                .progressViewStyle(.circular)
+                                .controlSize(.small)
+                        }
+                }
+            }
+            .cornerRadius(15)
+            .shadow(radius: 1)
+            .frame(width: 60, height: 60)
+            .padding()
+            Text("alert.app.delete")
+                .bold()
+                .frame(width: 250)
+        }
+        .padding()
+        .fixedSize()
+        .toolbar {
+            ToolbarItem(placement: .destructiveAction) {
+                if isClearingCache {
```

Comment:
This `if-else` statement is redundant, as the opacity is changed and has a `ProgressView` overlayed on the button text below.